tests/non-js-resources/subtests/image/subtests/data-url/parcel/index.md

```diff
@@ -1,6 +1,12 @@
 ---
-result: partial
-issue: https://github.com/parcel-bundler/parcel/issues/4320
+result: pass
+issue:
+  - url: https://github.com/parcel-bundler/parcel/issues/4320
+    fixedSince: 2.0.0-beta.1
+    status: closed
+  - url: https://github.com/parcel-bundler/parcel/issues/4869
```

Comment:
This issue is closed. Did you mean to mark it as such?

Reply:
Ah, it was fixed after I filed it.